src/components/slide/preview/slide-preview.jsx

```diff
@@ -0,0 +1,117 @@
+import { React, useEffect, useState } from "react";
+import { useParams } from "react-router";
+import { useTranslation } from "react-i18next";
+import { useDispatch } from "react-redux";
+import idFromUrl from "../../util/helpers/id-from-url";
+import {
+  api,
+  useGetV1SlidesByIdQuery,
+  useGetV1TemplatesByIdQuery,
+} from "../../../redux/api/api.generated";
+import { displayError } from "../../util/list/toast-component/display-toast";
+import RemoteComponentWrapper from "./remote-component-wrapper";
+import LoadingComponent from "../../util/loading-component/loading-component";
+
+/**
+ * Slide preview
+ *
+ * @returns {object} The slide preview
+ */
+function SlidePreview() {
+  const { t } = useTranslation("common");
+  const dispatch = useDispatch();
+  const [resources, setResources] = useState();
+  const [content, setContent] = useState();
+  const [mediaData, setMediaData] = useState();
+  const { id, templateId } = useParams();
+  const {
```

Comment:
Why all these loading functions?
All the data is available in slide form...

Reply:
Because it opens in a new tab, so I send the relevant ids and fetch the data. Figured it could be refreshed and work then.

Reply:
Then you will essentially have to do everything we do in slideManager again. That does not seem like a good solution.